.editorconfig

```
@@ -16,3 +16,6 @@ indent_size = 2
 
 [Makefile]
 indent_style = tab
+
+[*.sdc]
```

```suggestion
[*.scd]
```

And also a `tab_width=4`.